### ISSUE TITLE
Blank white app on opening Ente Auth v3.1.0 on Windows 11 24H2

### ISSUE THREAD

**Issue by 1aTa** [ente-io/ente#2674]
### Description

Open Ente Auth v3.1.0 on Windows 11 24H2 and presented with a blank white box.

![Screenshot 2024-08-12 163337](https://github.com/user-attachments/assets/e725af8f-5494-4106-bcf5-f0686f55b990)

Log: [2024-8-12.txt](https://github.com/user-attachments/files/16588322/2024-8-12.txt)

### Version

v3.1.0

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by pkosiec** [comment]
Same on Mac, but instead of white window there's black one:

<img width="902" alt="Screenshot 2024-08-12 at 16 29 46" src="https://github.com/user-attachments/assets/1bfe163a-65fc-4e41-ae2c-b0cb14c62c60">

**Comment by ImTou** [comment]
Same issue on Windows 10

**Comment by prateekmedia** [comment]
Sorry for the inconvenience, it has been resolved in latest release. Thanks for reporting!

**Comment by 1aTa** [comment]
Thank you, fixed in v3.1.1.

Disappointed to see that there's still no app lock on Windows.

This is a major security issue.

**Comment by prateekmedia** [comment]
@1aTa App Lock is available in Security->App Lock. I guess it is not available for system without proper biometrics / passcode for now.

**Comment by 1aTa** [comment]
Correct, on Windows it states "No system lock found".

Just a simple passcode would suffice.

**Comment by ljrafols** [comment]
Hey,
For whoever stumbles on this issue, I also encountered this issue but upgrading to the new version (3.1.1) did not fix it. Downgrading to a known working version also did not resolve it. 

After some finicking for the past few days I discovered deleting the relevant AppData folder `C:\Users\<USER>\AppData\Roaming\Ente Technologies, Inc\Ente Auth` will reset the application preferences will make it so the app loads again.
